bench/src/main/scala/org/http4s/bench/QueryParamBench.scala

```diff
@@ -0,0 +1,14 @@
+package org.http4s.bench
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+
+@BenchmarkMode(Array(Mode.AverageTime))
+@OutputTimeUnit(TimeUnit.MICROSECONDS)
+@State(Scope.Benchmark)
+class QueryParamBench {
+
+  @Benchmark def lensGet0(in: Nested0Input) =
```

Comment:
Did a commit get missed? This appears to be incomplete and is the source of the Travis failure.